documentation/source/user_section/tutorials/diffusion-weighted-mri/motion-correction-for-dmri.rst

```diff
@@ -21,8 +21,8 @@ To apply the algorithm, we use the ``sct_dmri_moco`` command:
 
```

Comment:
Bit of a snag here. I realize now that these changes need to correspond to changes made in the tutorial dataset, see: https://github.com/spinalcordtoolbox/sct_tutorial_data/pull/8

Once _that_ PR is merged, then a new dataset release needs to be created, which then needs to be updated in SCT:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/f9461b49a189cac7a647e0fac410c2390dd33ecd/documentation/source/conf.py#L100

So, I think this PR might need to first wait for https://github.com/spinalcordtoolbox/sct_tutorial_data/pull/8 to be merged first. :sweat_smile: 

(cc @mguaypaq)

Reply:
The updated commands in that PR should be identical to the updates commands in the tutorials, but... bit of a menial exercise to make sure the updates are the same, aha. :sweat:

Reply:
(Aside: It's possible that `batch_single_subject.sh` from the tutorial dataset should be stored in the SCT repo instead, which would make PRs like this a little easier, since all of the changes would happen in one place. This is a similar idea to https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3508.)

Reply:
Since the CI is passing in https://github.com/spinalcordtoolbox/sct_tutorial_data/pull/8, I think I'm going to just... go ahead and merge that PR as-is? The changes there are pretty much identical, so, I don't want to waste anyone's time here by nagging for an unnecessary review. :P